packages/next-admin/src/components/inputs/CheckboxWidget.tsx

```diff
@@ -12,10 +12,10 @@ const CheckboxWidget = ({
     <div className="relative flex items-start py-1">
       <div className="flex h-5 items-center">
         <input
-          defaultValue={value ? "on" : "off"}
+          value={value ? "on" : "off"}
+          onChange={() => value}
```

Comment:
Why this onChange ?

Reply:
Avoid the `value` without `onChange` warning, defaultValue was submitting erroneous data

Reply:
I've changed to use `readonly`